validation/validator.go

```diff
@@ -30,11 +38,39 @@ type failedValidator struct {
 	err error
 }
 
+// IsStringInSlice tells if a searched value is part of a slice or not
```

Comment:
I believe you used this function in other parts, no?

Reply:
You always need such a function.
Actually, it is used in keycloak-bridge for some validations